RecoLocalTracker/SiStripDataCompressor/plugins/SiStripDataCompressor.cc

```diff
@@ -0,0 +1,58 @@
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+#include "DataFormats/Common/interface/DetSetVector.h"
+#include "DataFormats/Common/interface/DetSetVectorNew.h"
+#include "DataFormats/SiStripCluster/interface/SiStripCluster.h"
+#include "DataFormats/SiStripDetSetCompressedCluster/interface/SiStripDetSetCompressedCluster.h"
+
+#include "RecoLocalTracker/SiStripDataCompressor/interface/SiStripCompressionAlgorithm.h"
+
+#include <vector>
+#include <memory>
+
+class SiStripDataCompressor : public edm::stream::EDProducer<> {
```

Comment:
it doesn't look like this can be a `stream` module - running with multiple streams results in
```
Module: SiStripDataCompressor:SiStripDataCompressor (crashed)
Module: none
Module: SiStripDataCompressor:SiStripDataCompressor
Module: SiStripDataCompressor:SiStripDataCompressor
Module: SiStripClusterizer:siStripClusters
Module: SiStripClusterizer:siStripClusters
Module: PoolOutputModule:outputRawLzma
Module: SiStripDataCompressor:SiStripDataCompressor
Module: SiStripClusterizer:siStripClusters
Module: SiStripClusterizer:siStripClusters
Module: PoolOutputModule:outputClustersLzma
Module: SiStripClusterizer:siStripClusters
Module: PoolOutputModule:outputClustersZlib
Module: SiStripClusterizer:siStripClusters
Module: SiStripClusterizer:siStripClusters
Module: SiStripDataCompressor:SiStripDataCompressor
```

Changing it to a `one` module works around the issue.